app/src/main/java/woowacourse/shopping/feature/login/LoginViewModel.kt

```
@@ -10,27 +12,29 @@ import woowacourse.shopping.util.SingleLiveData
-                    else -> _loginErrorEvent.postValue(LoginError.NotFound)
-                }
-            },
-            {
+        val basicKey = Authorization.getBasicKey(idValue, pwValue)
+
+        viewModelScope.launch {
+            try {
+                cartRepository.checkValidBasicKey(basicKey)
+                _loginSuccessEvent.postValue(basicKey)
+
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)